heron/spi/src/java/com/twitter/heron/spi/common/Keys.java

```diff
@@ -15,6 +15,8 @@
 package com.twitter.heron.spi.common;
 
 public class Keys {
+  public static final String SCHEDULER_PROPERTIES = "heron.scheduler.properties";
+  public static final String SCHEDULER_COMMAND_LINE_PROPERTIES_OVERRIDE_OPTION = "P";
```

Comment:
These two new keys are not consistent keys used below. Is there any particular reason for this inconsistency?